front_end/src/components/history/HistoryOverview.js

```diff
@@ -22,7 +23,8 @@ import ReviewRating from "./ReviewRating";
 import HistoryExpansionTable from "./HistoryExpansionTable";
 import "./history.css";
 import CancellationPrompt from "./CancellationPrompt";
-
+import ChangeReservation from "./ChangeReservation";
+import { Elements, StripeProvider } from "react-stripe-elements";
 const styles = {
   table: {
```

Comment:
I know this component wasn't made as part of this story but can you please remove these unused styles objects:
table
paddingDense
tableCell
dateChangedFrom
reviewFormHeaderText
ReviewButtonForPhone
historyHotelImageForPhone
HistoryPageTotalSmallForPhone

Thank you.

Reply:
Good catch! Removed and thank you